packages/css-library/src/stylesheets/formation-overrides/elements/typography.scss

```diff
@@ -87,7 +87,6 @@ h6 {
   font-family: tok.$font-family-serif;
   line-height: vars.$heading-line-height;
   margin-bottom: 0.5em;
-  margin-top: 1.5em;
```

Comment:
Removed this because it usually ends up being overridden in applications anyway.

In general, we should move to remove this file completely and instead use base/headings.scss because that uses uswds heading styles. The main thing in this rule that is still needed in vets-website is the margin-bottom rule, which several applications expect by default.